packages/react-toolbar/src/components/ToolbarDivider/useToolbarDividerStyles.ts

```diff
@@ -1,7 +1,15 @@
+import { SlotClassNames } from '@fluentui/react-utilities';
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { ToolbarDividerState } from './ToolbarDivider.types';
+import type { ToolbarDividerSlots, ToolbarDividerState } from './ToolbarDivider.types';
 
+/**
+ * @deprecated
+ */
 export const toolbarDividerClassName = 'fui-ToolbarDivider';
```

Comment:
For `ToolbarDivider` you can actually remove this class name instead of deprecating it since the component is not being exported from `@fluentui/react-components` yet:

```suggestion
```